implementations/elixir/ockam/ockam/mix.exs

```diff
@@ -14,7 +14,7 @@ defmodule Ockam.MixProject do
       version: @version,
       elixir: @elixir_requirement,
       consolidate_protocols: Mix.env() != :test,
-      elixirc_options: [warnings_as_errors: true],
+      elixirc_options: [warnings_as_errors: false],
```

Comment:
Intentional to include? Do we have any tracking issues to be able to restore it in the future?

Reply:
not intentional,  was a leftover :+1: